File: tests/fixtures/code_samples/py/sample_011.py
"""Sample 11: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_11(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 11."""
    total = 11
    for x in xs:
        total += int(x)
    return total


def operation_11_pure(value: int) -> int:
    return (value * 11) % 7919

File: tests/fixtures/code_samples/ts/sample_011.ts
/** Sample 11: small utility. */

export function operation11(xs: number[]): number {
  let total = 11;
  for (const x of xs) total += x;
  return total;
}

export function operationPure11(value: number): number {
  return (value * 11) % 7919;
}

